### DISCUSSION TITLE
pip-compile manager complains about `--allow-unsafe` and `--strip-extras`

### DISCUSSION THREAD

**Discussion by amezin** [renovatebot/renovate#27699] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Under "Repository problems" in Dependency Dashboard I now see:

* WARN: pip-compile: option --allow-unsafe not handled
* WARN: pip-compile: option --strip-extras not handled

These are valid pip-compile options, that previously were simply passed down to `pip-compile` utility

https://github.com/amezin/renovate-pip-compile

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: pip-compile: extracted command from header (branch="renovate/pip-compile-refresh")
{
  "fileName": "requirements/a.txt"
  "argv": [
    "pip-compile",
    "--allow-unsafe",
    "--strip-extras",
    "a.in"
  ]
  "isCustomCommand": false
}

WARN: pip-compile: option --allow-unsafe not handled (branch="renovate/pip-compile-refresh")
WARN: pip-compile: option --strip-extras not handled (branch="renovate/pip-compile-refresh")
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by viceice** [comment]
@not7cd Are those safe to allow?

**Reply by not7cd** [reply]
Yes, they are in allowed pip compile arguments list. But they are not used by this Renovate manager. I set a wrong logging level. 
There are two fixes I see:
1. Decrease logging level to DEBUG. 
2. Parse them and add them to internal structure.

But still this will not change managers current behaviour.

**Reply by not7cd** [reply, marked as answer]
Prepared a fix https://github.com/renovatebot/renovate/pull/27706
